scm/src/gmtb_scm_output.F90

```diff
@@ -219,6 +221,31 @@ subroutine output_init(scm_state)
   CALL CHECK(NF90_PUT_ATT(NCID=ncid,VARID=dummy_id,NAME="description",&
      VALUES="temperature tendency due to microphysics scheme"))
   CALL CHECK(NF90_PUT_ATT(NCID=ncid,VARID=dummy_id,NAME="units",VALUES="K s-1"))
+  CALL CHECK(NF90_DEF_VAR(NCID=ncid,NAME='dT_dt_ogwd',XTYPE=NF90_FLOAT,DIMIDS= (/ hor_dim_id, vert_dim_id, time_id /), &
```

Comment:
It would be nice if we could suppress the output of the 3d tendencies if ldiag3d/qdiag3d are false.

Reply:
What about if we just used a netCDF missing value if the logicals are false? Right now, it outputs zeros if they are false, but I suppose that could be misleading. I prefer to keep the variables actually in the output file so that scripts don't need to check for variables existing or not. I guess they would need to check for missing values instead though.